test/e2e/pd.go

```diff
@@ -87,11 +87,22 @@ var _ = Describe("PD", func() {
 		expectNoError(err, "Failed to create host1Pod")
 
 		By("waiting up to 60 seconds for host1Pod to start running")
-		expectNoError(waitForPodRunning(c, host1Pod.Name, 60*time.Second), "host1Pod not running after 60 seconds")
+		expectNoError(waitForPodRunning(c, host1Pod.Name, 120*time.Second), "host1Pod not running after 60 seconds")
 
 		By("deleting host1Pod")
 		expectNoError(podClient.Delete(host1Pod.Name), "Failed to delete host1Pod")
 
+		By(fmt.Sprintf("deleting PD %q", diskName))
+		err = deletePD(diskName, testContext.gceConfig.Zone)
+		for i := 0; i < 10; i++ {
```

Comment:
Can you flip the expiration loop over to this style? https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/kubectl.go#L149 Seems to fit fine.